src/lib.rs

```diff
@@ -337,7 +341,7 @@ impl core::fmt::Debug for Output {
         f.debug_struct("Output")
             .field("value", &self.value())
             .field("length", &self.length())
-            .field("is_begin", &self.is_begin())
+            .field("parent", &self.parent())
```

Comment:
I think this implementation can be replaced with `#[derive(Debug)]`.

Reply:
It's correct. Fixed. https://github.com/daac-tools/daachorse/pull/30/commits/c4663fc74a1d5415892702a3b5bdf550ac7dd562